src/main/jbake/content/blog/2016/09/19/hawkular-apm-on-msa.adoc

```diff
@@ -0,0 +1,163 @@
+= Using Hawkular APM on Red Hat\'s Microservices Reference Architecture example
+Juraci Paixão Kröhling
+2016-9-19
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, apm, msa
+
+The focus of the link:https://github.com/hawkular/hawkular-apm/releases/tag/0.10.0.Final[latest Hawkular APM release]
+has been in building a compatibility layer for ZipKin-aware applications. On this blog post, we'll show how Hawkular APM can
+be used as a drop-in replacement for ZipKin.
+
+Red Hat offers a link:http://developers.redhat.com/products/cdk/overview/[Container Developer Toolkit], or CDK, for its customers,
+allowing for a quick start on developing applications following the best practices on the microservices architecture.
+
+Along with the CDK, Red Hat publishes also a link:https://github.com/redhat-helloworld-msa["MSA - Hello World Microservices Architecture"],
+which demonstrates how applications might benefit from this architecture. It can also be used as a seed project for your own
```

Comment:
s/seed project/seed/ then , no :) Seed here being on this sense: https://github.com/angular/angular-seed